<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadR</class>
 <widget class="QDialog" name="uploadR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>45</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Realtime eQSL Upload</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRealtime">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>165</y>
     <width>266</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> Upload gesperrt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonESC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>276</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPasswd">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>115</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>239</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>239</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>75</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Passwd</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>210</y>
     <width>276</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Ist Realtime Upload freigegeben,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>230</y>
     <width>276</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>und Qslmodus auf eQsl gestellt,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>250</y>
     <width>256</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>wird das gelogte QSO sofort</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>270</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>an eQSL.cc gesendet.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
